src/test/pselect.c

```diff
@@ -0,0 +1,8 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "rrutil.h"
+
+int main(void) {
+  pselect(0, NULL, NULL, NULL, NULL, NULL);
```

Comment:
Thanks. That's what I get before I sent a diff before waiting for the tests to complete :-P